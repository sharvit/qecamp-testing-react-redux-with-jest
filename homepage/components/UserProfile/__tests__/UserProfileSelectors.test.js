import { testSelectorsSnapshotWithFixtures } from '../../../../testHelpers';
import {
  selectShowAvatar,
  selectShowPosts,
  selectShowPhotos,
} from '../UserProfileSelectors';

const state = {
  userProfile: {
    showAvatar: 'showAvatar',
    showPosts: 'showPosts',
    showPhotos: 'showPhotos',
  },
};

const fixtures = {
  'should select show-avatar': () => selectShowAvatar(state),
  'should select show-posts': () => selectShowPosts(state),
  'should select show-photos': () => selectShowPhotos(state),
};

describe('UserProfile - Selectors', () =>
  testSelectorsSnapshotWithFixtures(fixtures));
