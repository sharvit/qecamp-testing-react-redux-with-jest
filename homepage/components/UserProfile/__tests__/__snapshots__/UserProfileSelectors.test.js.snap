// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserProfile - Selectors should select show-avatar 1`] = `"showAvatar"`;

exports[`UserProfile - Selectors should select show-photos 1`] = `"showPhotos"`;

exports[`UserProfile - Selectors should select show-posts 1`] = `"showPosts"`;
